src/request.ts

```diff
@@ -35,6 +35,16 @@ const request: {
 	});
 });
 
+[ 'POST', 'PUT' ].forEach(method => {
+	Object.defineProperty(request, method.toLowerCase(), {
+		value(url: string, options: RequestOptions = {}): UploadObservableTask<Response> {
```

Comment:
We don't have a test for request options default to nothing, therefore we are missing a branch in our testing.